### PR TITLE
Add minimumRenewal, maximumRenewal and renewalPeriod to Renewal

### PR DESCRIPTION
**Pull request by ColinMaudry**
Closes https://github.com/open-contracting-extensions/european-union/issues/83

### CODE REVIEW

File: release-schema.json [hunk 1]
```diff
@@ -30,6 +30,27 @@
             "null"
           ],
           "minLength": 1
+        },
+        "minimumRenewal": {
```
Comment (jpmckinney) [inline release-schema.json]: Needs to be `minimumRenewals` and `maximumRenewals`

File: release-schema.json [hunk 2]
```diff
@@ -30,6 +30,27 @@
...
+        },
+        "maximumRenewal": {
+          "title": "Maximum renewal",
+          "description": "The maximum number of renewals. If there is no maximum, set to 1e9999 (which parses to infinity).",
+          "type": [
+            "number",
+            "null"
+          ]
+        },
+        "renewalPeriod": {
```
Comment (jpmckinney) [inline release-schema.json]: Should just be `period`, since it's already in scope of `Renewal` object.

Comment (ColinMaudry) [reply release-schema.json]: Done

File: release-schema.json (hunk 1 above)
Comment (ColinMaudry) [reply release-schema.json]: Done

**Pull request merged**